catroid/src/main/java/org/catrobat/catroid/io/StorageHandler.java

```diff
@@ -1417,14 +1417,11 @@ private void copyFile(InputStream in, OutputStream out) throws IOException {
 	}
 
 	public void unzipTemplate(String projectName, String templateName, String zipFileName, Activity activity) {
```

Comment:
There is also a UtilZip. All Zipfiles should be handled in the UtilZip or the StorageHandler. Not a few here, and a few there...